zipkin-collector-service/build.gradle

```diff
@@ -1,6 +1,13 @@
 apply plugin: 'application'
 mainClassName = 'com.twitter.zipkin.collector.Main'
 
+run {
+    // When configKey is dev, collector and query share ../zipkin.db
+    workingDir rootProject.projectDir
```

Comment:
so the root build directory isn't created by default.. I'll have to make sure it is.